db/migrate/20210325191756_add_published_to_testimonials.rb

```diff
@@ -0,0 +1,6 @@
+class AddPublishedToTestimonials < ActiveRecord::Migration[6.1]
+  def change
+    add_column :mentor_testimonials, :published, :boolean, null: false, default: true
+    add_column :mentor_testimonials, :deleted_at, :datetime, null: true
```

Comment:
Snuck in a little change too? :)

Reply:
Ha. This was where I started the whole thing! Then was like "fuck this".